firebase-ai/src/main/kotlin/com/google/firebase/ai/type/LiveSession.kt

```diff
@@ -325,21 +347,22 @@ internal constructor(
     audioHelper
       ?.listenToRecording()
       ?.buffer(UNLIMITED)
+      ?.flowOn(audioDispatcher)
       ?.accumulateUntil(MIN_BUFFER_SIZE)
       ?.onEach {
         sendMediaStream(listOf(MediaData(it, "audio/pcm")))
         delay(0)
       }
       ?.catch { throw FirebaseAIException.from(it) }
-      ?.launchIn(scope)
+      ?.launchIn(networkScope)
```

Comment:
shouldn't this be audio scope?

Reply:
No, because this corresponds to the network behavior. The `flowOn` ensures the actual recording and buffering occurs on the audio dispatcher, but everything after the `flowOn` occurs on the blocking dispatcher (via the `networkScope`).